src/com/theironyard/novauc/Children.java

```diff
@@ -0,0 +1,12 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by jerieshasmith on 2/7/17.
+ */
+public class Children {
+    int age;
+    String name;
+    int shoeSize;
+    String parent;
+    Boolean IsAlive;
+}
```

Comment:
the boolean variable on line 11 should be boolean isAlive. so let's clean up the variable naming a little.  Overall very nicely done.